morphic.js

```diff
@@ -3046,49 +3030,87 @@ Morph.prototype.fullCopy = function () {
     Other properties are also *shallow* copied, so you must override
     to deep copy Arrays and (complex) Objects
     */
-    var dict = {}, c;
-    c = this.copyRecordingReferences(dict);
+    var map = new Map(), c;
```

Comment:
This breaks <= IE10, and Safari 7. That's not a huge change, but not totally insignificant either, IMO.